prosody/Dockerfile

```diff
@@ -2,11 +2,19 @@ FROM jitsi/base
 
 RUN \
 	apt-dpkg-wrap apt-get update && \
-	apt-dpkg-wrap apt-get install -t stretch-backports -y prosody && \
+	apt-dpkg-wrap apt-get install -t stretch-backports -y prosody luarocks gcc libssl1.1 libssl1.0-dev git && \
 	apt-cleanup && \
 	rm -rf /etc/prosody
 
 COPY rootfs/ /
+COPY git-modules/jitsi-meet/resources/prosody-plugins /prosody-plugins
+
+RUN \
+	luarocks list --outdated && \
```

Comment:
I think I can remove this line because there are always no rocks installed at this time. Right?
Sorry I'm new in lua.

Reply:
I don't think it's needed in this setup.

Reply:
git-modules/jitsi-meet/resources/prosody-plugins is empty, the copy fails